Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -344,6 +344,12 @@ public async Task<bool> IsRoaming()
             string wifiIpV4 = null;
             string wifiIpV6 = null;
 
+            if (!matchingEngine.EnableEnhancedLocationServices)
```

Comment:
EnableEnhancedLocationServices will be sat in the Dll then?

Reply:
Just included the DLL with the fix to avoid the local network permissions.